### ISSUE TITLE
Problem with csquotes

### ISSUE THREAD

**Issue by EnraH** [ftilmann/latexdiff#139]
The attached tex file 
[latex-diff-mwe.txt](https://github.com/ftilmann/latexdiff/files/1727449/latex-diff-mwe.txt)
does not compile if line 31 is uncommented. There is a problem with csquote and latexdiff I suppose. The error message is 

! Extra }, or forgotten \endgroup.
\UL@stop ...z@ \else \UL@putbox \fi \else \egroup
                                                  \egroup \UL@putbox \fi \if...
l.31 ...some text \textquote[someone]{said this}.}
                                                  \DIFaddend

**Comment by ftilmann** [comment]
Try using option `--exclude-textcmd=textquote`  when running latexdiff.  All commands \text* are considered to be in the family \textrm \textbf  etc., which is clearly not justified for \textquote.
Else, using a different markup type, which does not use ulem package might also work (depends on the details of csquote package, which I don't know)

**Comment by EnraH** [comment]
Thanks for the reply. It seems to work. There is a similar problem with footnote. If the \DIFaddbegin\DIFadd{...}\DIFaddend encapsulate either \footnote or \textquote the compilation does not work. If however the encapsulation is swapped it works just fine.
[latex-diff-mwe.txt](https://github.com/ftilmann/latexdiff/files/1728218/latex-diff-mwe.txt)

**Comment by ftilmann** [comment]
\footnote should never be placed in the argument of \DIFadd as a result of latexdiff run. Instead I get something like
`\DIFaddbegin 

\footnote{\DIFadd{here is a footnote}}
\DIFaddend 
`
 If your MWE file in the last post really resulted from latexdiff run rather than being assembled by hand, then it is a bug. Please submit old and new input files and any options to latexdiff

**Comment by EnraH** [comment]
That's right. I fiddled around the latexdiff in order to understand the problem. Something is still not working. Now I run `latexdiff --exclude-textcmd=textquote difflatex_mwe_v1.tex difflatex_mwe_v2.tex > diff.tex` on difflatex_mwe_v1.tex
```\documentclass{article}
\usepackage{csquotes}

\title{Test}
\begin{document}
\maketitle
Some text.
\textquote[someone else]{bla}
Some other text.
\end{document}
```

against difflatex_mwe_v2.tex

```
\documentclass{article}
\usepackage{csquotes}

\title{Test}
\begin{document}
\maketitle
Some more text.
Some more other text.
\end{document}
```
with `difflatex --version` being `This is LATEXDIFF 1.2.1  (Algorithm::Diff 1.15 so, Perl v5.24.3)`. When compiling diff.tex there is the same problem as before. In the diff.tex the textquote commands are not being replaced. Also in the case where I first encountered the problem not all textquotes got replaced by mbox.
